### PR TITLE
JAVA-29330: Migrate spring-security-web-rest-custom to  parent-boot-3.

### PR DESCRIPTION
**Pull request by harry9656**

### CODE REVIEW

File: spring-security-modules/spring-security-web-rest-custom/src/main/java/com/baeldung/config/MainWebAppInitializer.java [hunk 1]
```diff
@@ -2,33 +2,26 @@
...
-     */
     @Override
-    public void onStartup(final ServletContext sc) throws ServletException {
+    public void onStartup(final ServletContext sc) {
         System.out.println("MyWebAppInitializer.onStartup()");
 
-        // Create the 'root' Spring application context
         final AnnotationConfigWebApplicationContext root = new AnnotationConfigWebApplicationContext();
         root.scan("com.baeldung.config.parent");
         // root.getEnvironment().setDefaultProfiles("embedded");
```
Comment (alvin577austria) [inline spring-security-modules/spring-security-web-rest-custom/src/main/java/com/baeldung/config/MainWebAppInitializer.java:27]: Do we know why we have this commented code here? Maybe we can remove this.

File: spring-security-modules/spring-security-web-rest-custom/src/main/java/com/baeldung/config/child/WebConfig.java [hunk 2]
```diff
@@ -27,23 +26,19 @@
 //@ImportResource({ "classpath:prop.xml" })
```
Comment (alvin577austria) [inline spring-security-modules/spring-security-web-rest-custom/src/main/java/com/baeldung/config/child/WebConfig.java:26]: Same as comment above

**Review by alvin577austria — CHANGES_REQUESTED** [review]

File: spring-security-modules/spring-security-web-rest-custom/src/main/java/com/baeldung/config/MainWebAppInitializer.java (hunk 1 above)
Comment (alvin577austria) [reply spring-security-modules/spring-security-web-rest-custom/src/main/java/com/baeldung/config/MainWebAppInitializer.java:27]: Like maybe it is mentioned in the article or so.

Comment (harry9656) [reply spring-security-modules/spring-security-web-rest-custom/src/main/java/com/baeldung/config/MainWebAppInitializer.java:27]: In fact, there is no mention. Therefore, removing.

File: spring-security-modules/spring-security-web-rest-custom/src/main/java/com/baeldung/config/child/WebConfig.java (hunk 2 above)
Comment (harry9656) [reply spring-security-modules/spring-security-web-rest-custom/src/main/java/com/baeldung/config/child/WebConfig.java:26]: It must be a consequence of a copy-paste from the ServiceConfig.class. I am removing them as well.

**Pull request merged**